Project.toml

```diff
@@ -8,6 +8,7 @@ Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 ArrayInterface = "4fba245c-0d91-5ea0-9b3e-6abc04ee57a9"
 BandedMatrices = "aae01518-5342-5314-be14-df237901396f"
 BlockBandedMatrices = "ffab5731-97b5-5995-9138-79e8c1846df0"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
```

Comment:
@pkj-m I forgot, could you add a version constraint in the [compat] section, to keep it to 0.17

Reply:
Done.